### PR TITLE
Fix SFPeoplePicker

### PR DESCRIPTION
**Pull request by Yopo-Nicolas**

### CODE REVIEW

**Review by martinweingart — CHANGES_REQUESTED** [review]

File: src/Components/SFPeoplePicker/SFPeoplePicker.tsx [hunk 1]
```diff
@@ -239,13 +273,23 @@ export const SFPeoplePicker = ({
     _event: React.ChangeEvent,
     newValue: SFPeopleOption
```
[the remark this answers is not in the document]
Comment (Yopo-Nicolas) [reply src/Components/SFPeoplePicker/SFPeoplePicker.tsx]: Done

File: src/Components/SFPeoplePicker/SFPeoplePicker.tsx [hunk 2]
```diff
@@ -211,53 +286,40 @@ export const SFPeoplePicker = ({
...
-        }
-      } else {
-        setAsyncOptions([]);
-      }
+  const renderTags = (value: SFPeopleOption[]): React.ReactNode => {
+    if (props.multiple) {
+      return (
+        <SFAutocompleteChipRender
+          disabled={disabled}
+          values={value.map((val) => val.name)}
```
Comment (martinweingart) [inline src/Components/SFPeoplePicker/SFPeoplePicker.tsx:294]: Missing `val` type

**Review by martinweingart — CHANGES_REQUESTED** [review]

File: src/Components/SFPeoplePicker/SFPeoplePicker.tsx (hunk 2 above)
Comment (Yopo-Nicolas) [reply src/Components/SFPeoplePicker/SFPeoplePicker.tsx]: Done

**Review by martinweingart — APPROVED** [review]
LGTM

**Pull request merged**